accord-core/src/main/java/accord/api/Query.java

```diff
@@ -9,5 +9,5 @@
      * Perform some transformation on the complete {@link Data} result of a {@link Read}
      * from some {@link DataStore}, to produce a {@link Result} to return to the client.
      */
-    Result compute(Data data);
+    Result compute(Data data, Read read, Update update);
```

Comment:
should `Update` have the `Nullable` annotation?  It is nullable on read-only queries

Reply:
Yep, happy to add that thanks.